cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -321,6 +321,49 @@
 }
 
 
+// Center content and bleed full-width images into gutters
+// Used on our story pages
+.content__center-image-bleed {
+
+    & > section {
+        margin: 0 auto;
+        max-width: 41.875rem;
+
+        .o-full-width-text-group {
+            overflow: inherit;
+        }
+
+        img.bleed-image {
+            @half-gutter-width: @grid_gutter-width / 2;
```

Comment:
Is this used?

Reply:
Yeah L342 and L343. The constant multiplication and division was visually overwhelming so I moved the basic gutter calcs to these variables.

Reply:
Oh I see. I had done a find search and didn't find it in the escaped string below. All good!